testing/path.go

```diff
@@ -66,6 +66,12 @@ func (path *Path) SetChannelOrdered() {
 	path.EndpointB.ChannelConfig.Order = channeltypes.ORDERED
 }
 
+func (path *Path) DisableUniqueChannelIDs() *Path {
+	path.EndpointA.disableUniqueChannelIDs = true
+	path.EndpointB.disableUniqueChannelIDs = true
+	return path
+}
+
```

Comment:
This is the opt-out function, currently used by only one test suite, and ideally it should not be used anymore.